src/test/regress/expected/multi_insert_select_conflict.out

```diff
@@ -515,19 +507,16 @@ SELECT * FROM target_table ORDER BY 1;
     10 |     0
 (10 rows)
 
--- prevent PG 11 - PG 12 outputs to diverge
-SET citus.enable_cte_inlining TO false;
 WITH cte AS(
 	SELECT col_1, col_2, col_3 FROM source_table_1
 ), cte_2 AS(
 	SELECT col_1, col_2 FROM cte
 )
 INSERT INTO target_table SELECT * FROM cte_2 ON CONFLICT(col_1) DO UPDATE SET col_2 = EXCLUDED.col_2 + 1;
 DEBUG:  distributed INSERT ... SELECT can only select from distributed tables
-DEBUG:  generating subplan XXX_1 for CTE cte: SELECT col_1, col_2, col_3 FROM on_conflict.source_table_1
-DEBUG:  generating subplan XXX_2 for CTE cte_2: SELECT col_1, col_2 FROM (SELECT intermediate_result.col_1, intermediate_result.col_2, intermediate_result.col_3 FROM read_intermediate_result('XXX_1'::text, 'binary'::citus_copy_format) intermediate_result(col_1 integer, col_2 integer, col_3 integer)) cte
-DEBUG:  Plan XXX query after replacing subqueries and CTEs: SELECT col_1, col_2 FROM (SELECT cte_2.col_1, cte_2.col_2 FROM (SELECT intermediate_result.col_1, intermediate_result.col_2 FROM read_intermediate_result('XXX_2'::text, 'binary'::citus_copy_format) intermediate_result(col_1 integer, col_2 integer)) cte_2) citus_insert_select_subquery
-DEBUG:  Collecting INSERT ... SELECT results on coordinator
+DEBUG:  CTE cte is going to be inlined via distributed planning
```

Comment:
Well, I'm not sure if this test is specifically need `MATERIALIZED` ctes, it seems the plan changes and it might matter. Maybe to these as materialized as well?